sass/_guardians.scss

```
@@ -0,0 +1,24 @@
+.guardian {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 20px;
+
+    .guardian-picture {
```

I think this rule could just be `img` since it's nested inside `.guardian`.

```suggestion
    img {
```